### PR TITLE
EA-2519: Attempt to fix XML serialization issues

### PR DESCRIPTION
**Pull request by ikattey**

### CODE REVIEW

File: entity-management-utils/src/main/java/eu/europeana/entitymanagement/serialization/EntityXmlSerializer.java [hunk 1]
```diff
@@ -41,7 +55,7 @@
 		"	  xmlns:ore=\"http://www.openarchives.org/ore/terms/\"\r\n" +
 		"         xmlns:skos=\"http://www.w3.org/2004/02/skos/core#\"\r\n" + 
 		"         xmlns:dcterms=\"http://purl.org/dc/terms/\" >";
-    	final String XML_HEADER_TAG_PLACE = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + 
+    	final String XML_HEADER_TAG_PLACE = "?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" +
```
Comment (gsergiu) [inline entity-management-utils/src/main/java/eu/europeana/entitymanagement/serialization/EntityXmlSerializer.java]: shouldn't this value start with < ?

File: entity-management-utils/src/main/java/eu/europeana/entitymanagement/serialization/EntityXmlSerializer.java [hunk 2]
```diff
@@ -50,7 +64,7 @@
 	    	"         xmlns:owl=\"http://www.w3.org/2002/07/owl#\"\r\n" + 
 	    	"         xmlns:skos=\"http://www.w3.org/2004/02/skos/core#\"\r\n" + 
 	    	"	  xmlns:ore=\"http://www.openarchives.org/ore/terms/\"\r\n" +
-	    	"         xmlns:dcterms=\"http://purl.org/dc/terms/\" >";
+	    	"         xmlns:dcterms=\"http://purl.org/dc/terms/\"";
```
Comment (gsergiu) [inline entity-management-utils/src/main/java/eu/europeana/entitymanagement/serialization/EntityXmlSerializer.java]: shouldn't this value end with > ?

File: entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/BaseEntity.java [hunk 3]
```diff
@@ -165,7 +165,6 @@ public void setIdentifier(String[] identifier) {
 		this.identifier = identifier;
 	}
 
-	@JsonIgnore
```
Comment (gsergiu) [inline entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/BaseEntity.java:168]: I think that JsonIgnore is still needed for the json serialization. Can we move the @JacksonXmlProperty annotation to the getEntityId() method?

**Review by gsergiu — COMMENTED** [review]
there are some inconsistencies in the constants for xml namespace mapping which I suspect will result in invalid xml response

**Comment by ikattey** [comment]
Review remarks incorporated.

Thanks @gsergiu!

**Pull request merged**